version

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
Why is a version file add what not used by python and why weren't the git tag used for versioning?

Reply:
python should be the next step with our own python action for all sc-apps
See: feature/OPS-1722-build_and_deploy_antivirus

Reply:
version file now is used by python the git tag uses the version text file for versioning.